snf/inject.h

```diff
@@ -9,20 +9,17 @@
 
 static inline int
 snf_inject_send_bulk(snf_inject_t inj, int timeout_ms, int flags,
-                     const uint8_t *pkts, uint32_t n_pkts, const uint32_t *lengths)
+                     const void **pkts, uint32_t n_pkts, const uint32_t *lengths)
```

Comment:
shouldn't it be `void * const *pkts`?

Reply:
I changed it to `uintptr_t *pkts` as you recommended below.